onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -478,4 +478,8 @@ MlasPlatformU8S8Overflow(
     return p.GemmU8U8Dispatch != p.GemmU8S8Dispatch;
 }
 
-#endif
\ No newline at end of file
+#endif
+
+thread_local size_t ThreadedBufSize = 0;
+thread_local std::unique_ptr<uint8_t, MlasAlignedDeleter> ThreadedBufHolder(nullptr,
```

Comment:
You will see crashes if the thread running this code lives longer than onnxruntime.dll(or libonnxruntime.so).

Reply:
what's your suggestion?

Reply:
No easy solution.  You could see we have a similar thing in CUDA EP, and we use a very complicated way to handle it. I will forward you an internal discussion about this issue.

Reply:
So ORT does not have a destructor?

Reply:
I might worry overmuch. We can continue the discussion when we hit a crash.

Reply:
Pushed another commit to simplify the deleter. now this object does not depend on anything ORT, all the code related to this object should be statically linked in standard c library, and all these objects do not depend on each other.